packages/plugin-pre/src/PreNode.ts

```diff
@@ -0,0 +1,7 @@
+import { VElement } from '../../core/src/VNodes/VElement';
+
+export class PreNode extends VElement {
```

Comment:
You can extend `VNode` rather than `VElement` because you're not using the `htmlTag` feature of `VElement` anyway. (And we will eventually remove `VElement` from the core anyway)

Reply:
It's because our rendering and parsing logic is outside of the node itself.